igcollect/linux_pressure.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+"""igcollect - PSI - Pressure Stall Information
+
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+import os
+import time
+
+from argparse import ArgumentParser
+
+
+def parse_args():
+    parser = ArgumentParser(prog='linux_pressure.py')
+    parser.add_argument('--prefix', default='pressure')
+    return parser.parse_args()
+
+
+def main():
+    if os.path.isdir('/proc/pressure'):
+        args = parse_args()
+        for pressure in get_pressure_stats():
+            print('{}.{}'.format(args.prefix, pressure))
+
+
+def parse_split_file(filename):
+    """Utility to read and parse a space delimited file"""
+    with open(filename, 'r') as fd:
+        return [line.strip().split(' ') for line in fd]
+
+
+def get_pressure_stats():
```

Comment:
Give the reviewer a comment about how that file or the lines looks like, so the reviewer is actually able to review the parsing process.

Reply:
I left a rather lengthy comment (stolen from the documentation) on what /proc/pressure/* is and how the data format looks like.

Reply:
Thx for the descriptive comment. The first part about the format is the most interesting for the reviewer. The second part looks more interesting for the whole script itself, for the single function a short description of how the data will be returned would be more interesting. JFTK.

I leave it resolved, not important for change.